src/entities/user.ts

```diff
@@ -0,0 +1,31 @@
+import {UserData} from './user.interface';
+
+export class User {
+    private _id: string;
```

Comment:
With TS we do not need underscore anymore, since you cannot call such prop anyway, so no need to mark it in such way.